README.md

```diff
@@ -148,18 +148,46 @@ Just run `helm uninstall --namespace tutor-multi tutor-multi` to uninstall this.
 ## Appendix B: how to create a cluster for testing on DigitalOcean
 
 If you use DigitalOcean, you can use Terraform to quickly spin up a cluster, try this out, then shut it down again.
-Here's how. First, put the following into `infra-tests/secrets.auto.tfvars` including a valid DigitalOcean access token:
+Here's how. First, put the following into `infra-examples/secrets.auto.tfvars` including a valid DigitalOcean access token:
 ```
 cluster_name = "tutor-multi-test"
 do_token = "digital-ocean-token"
 ```
 Then run:
 ```
-cd infra-example
+cd infra-examples/do
```

Comment:
nit: Don't you think it would be more straightforward to call it `infra-examples/digitalocean`?